src/common/reliability/Image-Segmentation/maskrcnn_resizeimage.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# coding: utf-8
+import sys
+import numpy as np
+from imgseg.predict import predict, show
+from torchvision.models.detection import maskrcnn_resnet50_fpn
+from PIL import Image
+import time
+# from IPython.display import display
+# from utils.ipstartup import *
+sys.path.insert(0, "utils")
+
+model = maskrcnn_resnet50_fpn(pretrained=True)
+
+# Applies MaskRCNN on downscaled image, by default the factor is 10x
+def predictByResize(image, flag=0, factor=10):
+    print("Original Image Dimension: ", image.size)
+    rimage = image
+    if flag == 1:
+        newsize = (int(image.size[0] / factor), int(image.size[1] / factor))
+        rimage = image.resize(newsize)
+    start_time = time.time()
+    print("After Resizing, Image Dimension: ", rimage.size)
+    out = predict(rimage, model)
+    print("Time: %s s" % (time.time() - start_time))
+    return (rimage, out)
+
+#File path
+def processImage(filepath):
```

Comment:
Please make all own function names in this PR snake_case instead of camelCase

```suggestion
def process_image(filepath):
```